### DISCUSSION TITLE
Use my own cloud

### DISCUSSION THREAD

**Discussion by rzaprianov** [flameshot-org/flameshot#3596] [Ideas]
Hello, 

Can I use my own cloud "Next cloud" instead of imgur? If yes - how?

**Comment by mmahmoudian** [comment]
It is not possible through Flameshot itself. When our plugin system gets implemented, then perhaps there can be a plugin for that. That said, you can use so many different ways to upload it to your Next Cloud, because at the end of the day Flameshot produces an image file. So you can use any tools you want to transfer the images to your NextCloud. For instance you can use [rclone](https://rclone.org/downloads/) and [configure it to work with your NextCloud instance](https://rclone.org/webdav/#nextcloud)
